src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    abstract double obtainArea();
```

Comment:
```suggestion
    double getArea();
```
it's abstract by default